packages/x-components/static-docs/experience-search-and-discovery/README.md

```diff
@@ -58,42 +59,15 @@ choose the modules and components that best suit your needs.
 
 :::
 
-## Try X Components to…
-
-- Surprise your shoppers by **rendering shopping-aimed search and discovery features** that
-  intuitively guide and inspire them with trends, suggestions, and recommendations, based on your
-  shoppers’ behavior.
-- Add a **predictive layer** to your search UI that gathers all search functionalities in one, from
-  autocomplete and autosuggest to ideas of what to look for next.
-- Delight your shoppers with an **elegant and multi-configurable SERP** to help them discover what
-  they’re exactly looking for.
-- Enhance the shopping experience by adding **additional features to the SERP** such as add-to-cart
-  buttons or product rating information.
-
-::: warning
-
-Some X&nbsp;Components are supported by Empathy Platform search and discovery features. If you’re
-using a different search engine, make sure it supports all the functionalities you need.
-
-:::
-
-::: interact
-
-Explore the Empathy Platform [interactive map](/explore-empathy-platform/diagram/interface/) for a
-sneak peak of the X&nbsp;Components available!
-
-:::
-
----
-
-## Discover Interface X Components
+#### Discover Interface X Components for web
 
 Check out these X&nbsp;Components for Vue.js applications. They’ll certainly come in handy if you
 want to make the difference!
 
 <CardCarousel :cards="[
     'x_popular_searches',
     'x_history_queries',
+    'x_my_history',
```

Comment:
Hi Luis! @mlcorton asked me this question.

This wouldn't work as a code example, but a custom component used in eDocs, that Vuepress will interpret directly as it is registered in the project.. It will look like this:

![Captura de pantalla 2022-08-29 a las 13 29 37](https://user-images.githubusercontent.com/45435602/187192069-f2b353dc-54e9-4ba5-b778-4d6e246abf99.png)

Most of these are listed in [eDocs README](https://github.com/empathyco/docs-empathy#card-carousel).

Reply:
Yeah! I supposed that @PedroOndh would explain it better than me ;) Thanks!

Reply:
Oh okey, perfect then!